src/controllers/update/update.manage.trusts.tell.us.about.the.former.bo.controller.ts

```diff
@@ -52,28 +55,30 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
       return res.render(UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_FORMER_BO_PAGE, pageProperties);
     }
 
-    const trustee = mapBeneficialOwnerToSession(req.body);
     const trusteeIndex = getTrusteeIndex(trust, trusteeId, TrusteeType.HISTORICAL);
 
     if (trust.HISTORICAL_BO && trusteeIndex >= 0) {
       // update existing trustee
-      trust.HISTORICAL_BO[trusteeIndex] = trustee;
```

Comment:
very similar code in src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts.
That does not have the comments which suggests they may not be needed or a suitably parameterized function could make it clearer. This rework could be a follow on.